adapter/persistence/src/main/java/org/anastaklo/portal/adapter/persistence/DocumentJpaEntity.java

```diff
@@ -0,0 +1,33 @@
+package org.anastaklo.portal.adapter.persistence;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Entity
+@Table(name = "document")
+public class DocumentJpaEntity extends BaseJpaEntity{
+    
+    @Column
+    @Getter
+    @Setter
+    @NonNull
+    private String label;
+
+    @Column
+    @Getter
+    @Setter
+    @NonNull
+    private String path;
+    
+    @ManyToOne(fetch = FetchType.LAZY)
+    @NonNull
+    @Getter
+    @Setter
+    private EventJpaEntity event;
```

Comment:
hm, i think a document - is the separate entity and an event should khow about own documents.
also some documents in the future will be not "link" with events.

Reply:
do you mean why I placed here `@ManyToOne`? I made it for better performance to prevent Hibernate from extra updates (https://vladmihalcea.com/the-best-way-to-map-a-onetomany-association-with-jpa-and-hibernate/)
I agree about linkage to events, so will remove `@NonNull`